### DISCUSSION TITLE
setGroupDescriptionis not working!

### DISCUSSION THREAD

**Discussion by imduuh** [open-wa/wa-automate-nodejs#2443] [Q&A] [answered]
I've been trying to use "client.setGroupDescription(group, description" for some time now, but it just doesn't return any errors, and there's no change in the group description either.

**Comment by smashah** [comment, marked as answer]
@imduuh Please make an issue for this.

Double check you are using a license as this is clearly stated to be a licensed feature:

https://docs.openwa.dev/classes/api_Client.Client.html#setGroupDescription
